tools/settings_template.py

```diff
@@ -1,5 +1,5 @@
-# This file will be edited (the {{{ }}} things), and then ~/.emscripten created
-# with the result, if ~/.emscripten doesn't exist.
+# This file will be edited (the {{{ }}} things), and written to `.emscripten`
+# when emscripten is direct uses and no config file is found.
```

Comment:
i don't understand this sentence. "direct uses" - what is "direct" referring to?

could this be `when emscripten is used and no config file is found` ?